src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -252,11 +253,15 @@ def fetch_codecov_data(
                     repo,
                     branch,
                 )
+                ref_source = "from_git_blame"
             except Exception:
                 logger.exception(
                     "Failed to get commit sha from git blame, pending investigation. Continuing execution."
                 )
 
+        with configure_scope() as scope:
+            scope.set_tag("codecov.ref_source", ref_source)
```

Comment:
I'm moving the tagging here to not pass `has_error_commit` around between functions.